rero/reroils-app
# feat: add icons by doc type

Signed-off-by: Aly Badr <aly.badr@rero.ch>
Co-authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

How to test:
- git reroils_app PR#65 and reroils_data PR#44
- invenio collect -v
- run_tests.sh
- place any bibliographic search and make sure icons are present by doc type in the Full View and Brief View
